cores/esp32/esp32-hal-rgb-led.h

```diff
@@ -11,8 +11,19 @@ extern "C" {
 #define RGB_BRIGHTNESS 64
 #endif
 
+typedef enum {
+  RGB,
+  BGR,
+  BRG,
+  RBG,
+  GBR,
+  GRB
+} color_order_t;
```

Comment:
naming it so simply is generally not a great idea, because it would clash with any other lib out there  that might have gone the same way. using enum like `rgb_led_color_order_t` and values like `LED_COLOR_RODER_RGB` would lower the possibility of such clash to a minimum.

Reply:
you're right, done it.